src/Features/RoyalShaving/components/StageContent/StageContent.tsx

```diff
@@ -0,0 +1,35 @@
+import { PropsWithChildren, ReactElement } from 'react';
+
+type ContentPositionVariant = 'contentLeft' | 'contentRight';
+type HeadlinePositionVariant = 'headlineLeft' | 'headlineRight';
+
+interface StageContentprops extends PropsWithChildren {
+  contentPosition: ContentPositionVariant;
+  headlinePosition: HeadlinePositionVariant;
+  stageHeadline: string;
+}
+
+export const StageContent = ({
+  contentPosition,
+  headlinePosition,
+  stageHeadline,
+  children,
+}: StageContentprops): ReactElement => {
+  const classes = {
```

Comment:
use the same approach as Ainura did after my review: https://github.com/Alex-2kZharkov/oldboy-barbershop/pull/36#discussion_r1326786149

Reply:
I pulled the changes from development, and there's no any mistakes now